backend/src/routes/report.route.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { authMiddleware } from '../middleware/auth.middleware';
+import {
+	createReportController,
+	listReportsController,
+	updateReportStatusController,
+} from '../controller/report.controller';
+
+const router = Router();
+
+router.get('/', listReportsController);
```

Comment:
this one should check for session also because this is internal api for admin and user to see list of report


```suggestion
router.get('/', authMiddleware, listReportsController);
```